rust/src/sdp/logger.rs

```diff
@@ -65,6 +62,22 @@ pub fn sdp_log(msg: &SdpMessage, js: &mut JsonBuilder) -> Result<(), JsonError>
     Ok(())
 }
 
+fn log_time_description(
+    time: &Vec<TimeDescription>, js: &mut JsonBuilder,
+) -> Result<(), JsonError> {
+    js.open_array("time_descriptions")?;
```

Comment:
Does the array have always one element ?
We do not want to open and log empty array

Reply:
I guess it does because of `many1` right ?